challenge01/solution-josebrodriguez.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# solution-josebrodriguez.sh
+
+# Created by: Jose Rodriguez on 10/04/2021
+
+FILENAME="input.txt"
```

Comment:
To pass the automated test, you should read from `stdin` rather than an input file.